browser/actors/FathomChild.jsm

```diff
@@ -13,6 +13,8 @@ var EXPORTED_SYMBOLS = ["FathomChild"];
 //   "resource://gre/modules/Fathom.jsm"
 // );
 
+const { Fathom } = ChromeUtils.import("resource:///modules/Fathom.jsm");
```

Comment:
Re: your commit comment:
> Should we use import or defineModuleGetter? Unclear.

I looked up the difference between [`import`](https://searchfox.org/mozilla-central/source/dom/chrome-webidl/ChromeUtils.webidl#314) and [`defineModuleGetter`](https://searchfox.org/mozilla-central/source/dom/chrome-webidl/ChromeUtils.webidl#347); it appears the latter is done lazily. This is a good option if we are not using a module right away and/or the module is not guaranteed to be used in the main code path (e.g. say if we only use the module in a particular case and not always).

Reply:
What do you know about the execution model for these .jsm files? Do they get instantiated once and anything loaded (whether eagerly or lazily) hangs around then forever? Or do they get torn down at some point?

Reply:
Hm... I've never thought about it before, but based on the source code above and the docs for [`ChromeUtils.unload`](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Language_Bindings/Components.utils.unload)[1], the module is imported once, then cached and likely unloaded by Firefox either as part of shutdown or when nothing is referencing the module anymore. Beyond that I don't know when/how it happens.

[1]:
> You should not unload modules that are part of the application (anything under resource://gre/ or resource:/// or resource://app/) as these may be in use by other extensions and the application.

Reply:
Reading the docs for `import` and `defineModuleGetter`, I see they make clear that the imported module gets evaluated once and stored off to the side in a hash table, whatever happens to the importing module. Let me see if I can get it working lazily; it's probably more neighborly to defer as much work as possible as long as possible. The downside is it may make our benchmarking harder, since the first time will be slower.

Reply:
There, that works. Any reason not to merge?

Reply:
Seems good to me.